src/containers/PaymentInfo/PaymentInfo.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable react/forbid-prop-types */
+
+import React, { useEffect } from 'react';
+import { withTranslation } from 'react-i18next';
+import labeling from 'containers/labeling';
+import PaymentMethod from 'components/PaymentMethod';
+import SectionHeader from 'components/SectionHeader';
+import Transactions from 'components/Transactions';
+import { PropTypes } from 'prop-types';
+import UpdatePaymentDetailsPopup from 'components/UpdatePaymentDetailsPopup';
+import { useSelector, useDispatch } from 'react-redux';
+import GracePeriodError from 'components/GracePeriodError';
+import { init as initPublisherConfig } from 'redux/publisherConfigSlice';
+import withAddPaymentDetailsFinalizationHandler from 'containers/WithAddPaymentDetailsFinalizationHandler';
+import { updatePaymentDetailsPopup } from 'redux/popupSlice';
+import { WrapStyled } from './PaymentInfoStyled';
+
+const PaymentInfoFn = ({
+  adyenConfiguration: adyenConfigurationProp,
+  t,
+  displayGracePeriodError
+}) => {
+  const dispatch = useDispatch();
+
+  const { adyenConfiguration: adyenConfigurationStore } = useSelector(
+    state => state.publisherConfig
+  );
+
+  const popupManager = useSelector(state => state.popupManager);
```

Comment:
const {paymentDetails: paymentDetailsPopup} = useSelector(state => state.popupManager);
and then in l:52
paymentDetailsPopup.isOpen ? ....